.github/workflows/pull_request.yaml

```diff
@@ -9,7 +9,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [18]
+        node-version: [21]
```

Comment:
do we drop node 18 support? If not, can we test on both?

Reply:
> do we drop node 18 support? If not, can we test on both?
Yes, Reason: [javascript-engineenvironment-support](https://github.com/qdrant/openapi-typescript-fetch?tab=readme-ov-file#javascript-engineenvironment-support)

Reply:
Node 18 is LTS and is still supported. We can't just drop it

Reply:
As suggested we skip BigInt tests for Node < 21, so we can come back to Node 18 compat 👍🏼